From c6819380f27eeb57210b3d2758aa2f8e6af54b56 Mon Sep 17 00:00:00 2001
From: Chong Lu <Chong.Lu@windriver.com>
Date: Tue, 3 Jun 2014 09:56:50 +0800
Subject: [PATCH] nfs-utils: unset CFLAGS for testlk build

testlk is built with host gcc at do_compile stage, which leads to unrecognized
some flags for special architecture. So unset CFLAGS for testlk to make sure
it passed.

Upstream-Status: Pending

Signed-off-by: Chong Lu <Chong.Lu@windriver.com>
---
 tools/locktest/Makefile.am |    1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/locktest/Makefile.am b/tools/locktest/Makefile.am
index 3156815..325b061 100644
--- a/tools/locktest/Makefile.am
+++ b/tools/locktest/Makefile.am
@@ -1,6 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 
 CC=$(CC_FOR_BUILD)
+CFLAGS =
 LIBTOOL = @LIBTOOL@ --tag=CC
 
 noinst_PROGRAMS = testlk
-- 
1.7.9.5

